### ISSUE TITLE
[New Feature]: feedback page addition

### ISSUE THREAD

**Issue by thesri** [GameSphere-MultiPlayer/PACKABUNCHAS#5]
### 🎮 feature Request

feature logic and basic description
customer rating or feedback will be included in this 

### Point down the features

feature points
will have qs like how would they rate this and all and ask for any difficulties

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by Durgesh4993** [comment]
@thesri Could you please explain more what you want to do, if possible you can also add screenshot

**Comment by Durgesh4993** [comment]
If you have any doubt you can directly as on our discord (https://discord.gg/rZb46cCMmK), we have created seprate chennals for all projects...

Thank you :)

**Comment by prajwal9773** [comment]
Hi, sir can you assign this to me, I can contribute

**Comment by thesri** [comment]
> @thesri Could you please explain more what you want to do, if possible you can also add screenshot

Hey yes I'll add the reference![image](https://github.com/GameSphere-MultiPlayer/PACKABUNCHAS/assets/98333833/8e6dffc5-c1b3-42fe-9a4f-8ee291b8bcfa)
Something like this??

**Comment by prajwal9773** [comment]
yes, I can add the above functionality, like the reference given

**Comment by thesri** [comment]
> yes, I can add the above functionality, like the reference given

I'm a Mentee:) chill out

**Comment by Hacxmr** [comment]
@Durgesh4993 I'm interested in contributing to this issue. Kindly assign me this issue under GSSoC!

**Comment by mittal246** [comment]
@thesri kindly assign this task to me .

**Comment by Durgesh4993** [comment]
@prajwal9773 @Hacxmr @mittal246 Please raise any other issue,  i'll assign you

**Comment by Durgesh4993** [comment]
If you have any doubt you can directly send message on our discord (https://discord.gg/rZb46cCMmK), we have created seprate chennals for all projects...

Thank you :)

**Comment by somiljainspace** [comment]
Describe the bug ✍️

I would like to add a feedback page in the game in the theme of your existing game that would rate the game from 0-10 and also the players can add a comment if they find any bug which will be reflected in a database 


Select program in which you are contributing

GSSoC24

Code of Conduct

 I follow [CONTRIBUTING GUIDELINE](https://github.com/ChromeGaming/PACKABUNCHAS/issues/5) of this project.
